### PR TITLE
[CN-448] Update WAN replication page and sample yaml

### PR DESCRIPTION
**Pull request by cagric0**

### CODE REVIEW

File: docs/modules/ROOT/pages/wan-replication.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 = Configuring WAN Replication
-:description: pass:q[WAN Replication allows you to keep Hazelcast clusters in sync across multiple sites or cloud deployments. In Hazelcast Platform Operator, you can create a `WanReplication` resource to replicate a map from a source to a target Hazelcast cluster. ]
+:description: pass:q[WAN Replication allows you to keep Hazelcast clusters in sync across multiple sites or cloud deployments. In Hazelcast Platform Operator, you can create a `WanReplication` resource to replicate multiple maps from multiple source clusters to a single target Hazelcast cluster. ]
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/wan-replication.adoc:2]:
```suggestion
:description: pass:q[WAN replication allows you to keep Hazelcast clusters in sync across multiple sites or cloud deployments. In Hazelcast Platform Operator, you can create a `WanReplication` resource to replicate multiple maps from multiple source clusters to a single target Hazelcast cluster. ]
```

File: docs/modules/ROOT/pages/wan-replication.adoc [hunk 2]
```diff
@@ -21,8 +21,11 @@ You can use the following configuration options in your `WanReplication` resourc
 |===
 |Field|Description
 
-|`mapResourceName`
-|The name of the custom `Map` resource to which WAN replication applies.
+|`resources`
+|The list of the custom `Map` or `Hazelcast` resource to which WAN replication applies.
+
+- `name`: The name of custom resource to which WAN replication applies.
+- `kind`: The kind of custom resource. Either `Map`(default) or `Hazelcast`. The kind `Hazelcast` leads to apply WAN replication to the all custom maps created under the given Hazelcast resource.
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/wan-replication.adoc:28]:
```suggestion
- `kind`: The type of custom resource. Either `Map`(default) or `Hazelcast`. Use the  `Hazelcast` kind to apply WAN replication to all custom maps created under the given `Hazelcast` resource.
```

**Review by angelasimms — CHANGES_REQUESTED** [review]
Just a couple of minor changes.

File: docs/modules/ROOT/pages/wan-replication.adoc [hunk 3]
```diff
@@ -21,8 +21,11 @@ You can use the following configuration options in your `WanReplication` resourc
 |===
 |Field|Description
 
-|`mapResourceName`
-|The name of the custom `Map` resource to which WAN replication applies.
+|`resources`
+|The list of the custom `Map` or `Hazelcast` resource to which WAN replication applies.
```
Comment (angelasimms) [inline docs/modules/ROOT/pages/wan-replication.adoc:25]:
```suggestion
|The list of custom `Map` or `Hazelcast` resources to which WAN replication applies.
```

**Review by angelasimms — CHANGES_REQUESTED** [review]
Sorry, just one final change. My mistake.

**Review by angelasimms — APPROVED** [review]
All looks good to me. Thank you!

File: docs/modules/ROOT/pages/wan-replication.adoc [hunk 4]
```diff
@@ -67,7 +70,7 @@ After creating the `WanReplication` resource, run the following command to find
 
 [source,shell]
 ----
-kubectl get wanreplication wanreplication-sample -o wide
+kubectl get wanreplication wanreplication-sample
```
Comment (mtyazici) [inline docs/modules/ROOT/pages/wan-replication.adoc:73]:
```suggestion
kubectl get wanreplication wanreplication-sample -o wide
```

Or we can remove the `message` column from the output of this command.

**Review by mtyazici — APPROVED** [review]

**Pull request merged**